src/ChipsCatalog.js

```diff
@@ -91,6 +91,15 @@ class ChipsDemos extends Component {
     );
   }
 
+  renderChipWithCancel(text, classes) {
```

Comment:
I'm inclined to leave this out - IIUC this is only intended to be used by input chips, and we don't yet have an input chips demo. AFAICT the chips article doesn't show remove icons within filter chips.

I'd be inclined to model the shaped chips after the action chips example.

Reply:
Removed cancel icon from the chip.